apps/api/src/app/topics-v2/usecases/upsert-topic/upsert-topic.usecase.ts

```diff
@@ -44,4 +47,14 @@ export class UpsertTopicUseCase {
       created: !topic,
     };
   }
+
+  private isValidTopicKey(key: string): void {
+    if (VALID_ID_REGEX.test(key)) {
+      return;
+    }
```

Comment:
should we reuse the same function we have in CreateTopic usecase here?

Reply:
Create topic is actually going to be deprecated, so not sure if it's needed

Reply:
(it's v1)